django/db/models/sql/compiler.py

```diff
@@ -1627,6 +1630,9 @@ def explain_query(self):
                 else:
                     yield value
 
+    def comments_sql(self):
+        return [f"/* {comment} */" for comment in self.query.comments]
```

Comment:
another idea: perhaps we can add the `/* ... */` wrapping to comments in `QuerySet.comment()`. Then we wouldn't even need a `comments_sql` method, the tuple can just be thrown into the SQL list.

Reply:
Done